### PR TITLE
Favorite opponent rival

### PR DESCRIPTION
**Pull request by wally-yawn**

### CODE REVIEW

File: data/games_test_3.csv
Comment (Crosswolfv1) [inline data/games_test_3.csv:1]: cool a new test scheme good for not potentially breaking other tests

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -462,6 +462,78 @@ def biggest_team_blowout(team_id)
...
+      end
+    end
+ 
+    away_team_games.each do |game|
+      if !opponents.include?(game.home_team_id)
+        opponents << game.home_team_id
+      end
+    end
+  
+    # require 'pry'; binding.pry
```
Comment (Crosswolfv1) [inline lib/stat_tracker.rb]: Please remove this pry binding prior to merge

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -462,6 +462,78 @@ def biggest_team_blowout(team_id)
       diff = (game.home_goals - game.away_goals).abs
       biggest_blowout = diff if diff > biggest_blowout
     end
-    biggest_blowout
+    biggest_blowout  end
```
Comment (Crosswolfv1) [inline lib/stat_tracker.rb]: Is this formatting right or is github being weird?

**Review by Crosswolfv1 — CHANGES_REQUESTED** [review]

File: lib/stat_tracker.rb (hunk 1 above)
Comment (wally-yawn) [reply lib/stat_tracker.rb]: removed

File: lib/stat_tracker.rb (hunk 2 above)
Comment (wally-yawn) [reply lib/stat_tracker.rb]: I didn't touch that so probably resolving a merge conflict did it, updated

**Review by Crosswolfv1 — APPROVED** [review]
No longer see earlier issues, approved for merge

**Review by Montana-Pfeifer — COMMENTED** [review]
all methods look good and all changes from prior review have been fixed.

**Pull request merged**